scenarios/bin/parse_compose_services

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
All files under bin/ uses dashes instead of underscores, please rename this to `parse-compose-services`